modules/social_features/social_profile/social_profile.post_update.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Social Profile.
+ */
+
+use Drupal\profile\Entity\ProfileInterface;
+
+/**
+ * Update Profile names.
+ */
+function social_profile_post_update_0001(&$sandbox) {
```

Comment:
/s `social_profile_post_update_0001 ` to `social_profile_post_update_profile_names_update_10101`

Reply:
I see that in OS mostly NAME follows by number. 

- https://github.com/goalgorilla/open_social/blob/1980f5a97a71ed5e7ba86bc4e49941c51110033f/modules/custom/activity_creator/activity_creator.post_update.php#L152
- https://github.com/goalgorilla/open_social/blob/1980f5a97a71ed5e7ba86bc4e49941c51110033f/modules/social_features/social_core/social_core.post_update.php#L20


So I think then will be better to be in the same style:
`social_profile_post_update_10101_profile_names_update`

Reply:
Right! Well spotted!

Reply:
Updated.